NEWS.org

```diff
@@ -198,7 +198,7 @@
   Fixed highlight color  of annotated text that starts  from the first
   character of the buffer's content.
 
-- 2021-02-05 V1.1.5 Bastian Bechtold, cage ::
+- 2021-03-17 V1.1.5 Bastian Bechtold, cage ::
```

Comment:
Why was the date changed?

Reply:
Hi!

I made an error updating NEWS file from 1.1.4 to 1.1.5, i copied the entry from the former to write the latter but i forgot to change the release date that was, in fact, duplicated (you can check the file NEWS on the master branch, 1.1.4 and 1.1.5 share the same release date). I was able to find a plausible release date from commit log messages and used to fix the 1.1.5 entry.

Bye!
C.